cmd/manager/collection/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM registry.access.redhat.com/ubi7/ubi-minimal:latest
+
+ENV OPERATOR=/usr/local/bin/kabanero-collection-operator \
```

Comment:
So, when I did the admission webhook, I removed a lot of this.  I didn't do anything with `USER_UID` or `USER_NAME` and I did not call `user_setup` or use `/usr/local/bin/entrypoint` as the entrypoint.  I just called the compiled go program directly.  I think this may solve your problem around exporting `ENV OPERATOR`.